src/modules/integrator-guide/pages/authentication/external/index.adoc

```diff
@@ -97,7 +97,6 @@ To configure role membership, click on the edit icon next to the external authen
 ----
 
 On the bottom of the modal you should see three inputs for permitted, prohibited, and forced roles.
-//Editor's note: what is a modal? I have never seen it appear in documentation before, so see what that is about. Does it need to be elaborated, is it a mistake, does the average reader know what it is?
```

Comment:
That is the little window that shows over the current content; usually seen when you are opting out of cookies

Reply:
I'll get a grip eventually. Thanks.